apps/pipelines/nodes/nodes.py

```diff
@@ -63,7 +71,8 @@ def all_variables(in_):
             # As a last resort, just set the all the variables in the template to the input
             content = all_variables(input)
         template = SandboxedEnvironment().from_string(self.template_string)
-        return template.render(content)
+        output = template.render(content)
+        return PipelineState(messages=[output], outputs={node_id: output})
```

Comment:
could be nice to make this a classmethod:

```python
def for_node_output(node_id, output)
   ...
```

Reply:
https://github.com/dimagi/open-chat-studio/pull/869/commits/eb528cb4b649bfe8e5192d71a4982bc0a62d7fcf